lock/client/client.go

```diff
@@ -0,0 +1,136 @@
+// Package client implements a client to the volmaster to acquire locks.
+package client
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/contiv/volplugin/config"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+// Driver is the main force behind the lock module, it controls all methods and
+// several variables.
+type Driver struct {
+	master         string
+	mountStopChans map[string]chan struct{}
+	mountMutex     sync.Mutex
+}
+
+// NewDriver constructs a new driver.
+func NewDriver(master string) *Driver {
+	return &Driver{
+		master:         master,
+		mountStopChans: map[string]chan struct{}{},
```

Comment:
minor: I think `make( map[string]chan struct{})` might be more Go-like instantiation of an empty map?